lib/methods/query.js

```diff
@@ -34,7 +34,24 @@ exports.query = function (Model, options, callback) {
 		whereQuery = this.translateWhereToQuery(where, values);
 	}
 
-	var order = Model.translateKeysForPayload(options.order);
+	/**
+	 * Transform stirng property to default orderTo clause
+	 * @param {any} prop
+	 * @returns {Object}
+	 */
+	function propToOrder(prop) {
+		var order = {};
+		order[prop] = 1;
+		return order;
+	}
+
+	var order = _([options.order])
+		.map(function (orderBy) {
```

Comment:
Any particular reason you wrap this in a lodash object and use map? You manually create an array with only one entry so the map will only be executed on that one entry anyway. I was wondering if something like this would also work or if i'm missing anything here?
```javascript
var order = options.order;
if (typeof options.order === 'string') {
	order = {};
	order[options.order] = 1;
}
var order = Model.translateKeysForPayload(options.order);
```